Ed: Fix GTFS parse and use SuspendedCableCar (#2591)
hove-io/navitia: documentation/slate/source/includes/apis.md

| nop     | max_duration         | int     | Maximum duration of journeys in seconds (from `datetime` parameter).<br>More usefull when computing an isochrone (only `from` or `to` is provided).<br>On a classic journey (from-to), it will mostly speedup Navitia: You may have journeys a bit longer than that value (you would have to filter them).    | 86400       |
| nop     | disruption_active    | boolean | For compatibility use only.<br>If true the algorithm take the disruptions into account, and thus avoid disrupted public transport.<br>Rq: `disruption_active=true` = `data_freshness=realtime` <br>Use `data_freshness` parameter instead       |  False     |
| nop     | wheelchair           | boolean | If true the traveler is considered to be using a wheelchair, thus only accessible public transport are used<br>be warned: many data are currently too faint to provide acceptable answers with this parameter on       | False       |

> | nop     | direct_path          | enum    | Specify if direct paths (fallback modes only are used) should be suggested.<br>possible values: <ul><li>`indifferent`</li><li>`none` to have only journeys using some PT</li><li>`only` to have only journeys without PT</li></ul>      | indifferent |

Reviewer: Or maybe simply :
```suggestion
| nop     | direct_path          | enum    | Specify if direct paths should be suggested (using fallback modes only).<br>possible values: <ul><li>`indifferent`</li><li>`none` for journeys using PT only</li><li>`only` for journeys without PT</li></ul>      | indifferent |
```
